docs/en/test_cards/mc_07_optical_flow_failure.md

```diff
@@ -0,0 +1,44 @@
+# Test MC_07 - Optical Flow Failure
+
+## Objective
+
+To test that optical flow works as expected with a low mounted optical flow sensor
+
+## Preflight
+
+Ensure that the drone's optical flow sensor is mounted less than an inch off of the ground.
+
+Disconnect all GPS / compasses and ensure vehicle is using optical flow for navigation
+([Setup Information here](../sensor/optical_flow.md))
+
+Ensure there are no other sources of positioning besides optical flow
+
+- [EKF2_OF_CTRL](../advanced_config/parameter_reference.md#EKF2_OF_CTRL): `1`
+- [EKF2_GPS_CTRL](../advanced_config/parameter_reference.md#EKF2_GPS_CTRL): `0`
+- [EKF2_EV_CTRL](../advanced_config/parameter_reference.md#EKF2_EV_CTRL): `0`
+- [SYS_HAS_MAG](../advanced_config/parameter_reference.md#SYS_HAS_MAG): `0`
```

Comment:
flow doesn't provide heading, why disable mag?

Reply:
we see this configuration for clients who are flying indoors and cant trust their mag. is there a mag natively on the ark FCU?

Reply:
I didn't think you could takeoff without a source of heading. Yeah all our FCs have mags

Reply:
well, might be a good idea to add that and try again. because we can takeoff, but the heading is north on bootup.